README.md

```diff
@@ -2086,6 +2086,24 @@ function quux ({a, b}) {
 
 }
 
+/**
+ * @param foo
+ * @param foo.a
+ * @param foo.b
+ */
+function quux ({"a": A, b}) {
+
+}
+
+/**
+ * @param foo
+ * @param foo.a-b
```

Comment:
I think we have a slight problem here. I wanted to add this as `@param foo["a-b"]` because this is the valid JS syntax, but then the tests would not pass. `foo.a-b` is syntactically valid but not semantically (cause it's a subtraction operation 😅 )

Reply:
How about `foo."a-b"`? I think that is the jsdoc namepath way and seems to work in https://jsdoctypeparser.github.io/ ...